templates/elements/quote/quote.html.twig

```diff
@@ -1,8 +1,8 @@
 <div class="decanter-quote">
-  <img class="decanter-quote__img " src="../../../img/circle.png" alt="Please provide alt text">
+  <img class="decanter-quote__img " src="../../../img/{{ image }}" alt="{{ altText }}">
```

Comment:
@JBCSU Let's change the src attribute to `src="{{ image }}"` so we can supply full paths to this template instead of assuming `../../../img/`

Reply:
Done.